docs/tutorials/index.html

```diff
@@ -4,30 +4,35 @@
 include_masonry: "true"
 ---
 
-{% assign tutorials=site.pages | where: "category", "tutorials" | sort: "subject" | group_by: "subject" %}
+<p class="lead">
+    These tutorials will walk you through working with a wide range of hardware and
+    software. If you're looking for help with a specific goal, this is the place to
+    look!
```

Comment:
Are tutorials meant only for hardware and software or are other concepts welcome? 
This dertermines also what people will think they can contribute.

I mean for example:
Robotics - Odometry
Robotics - Motion Control
Kalman Filters, SLAM, etc.


E.g. with ev3dev example realization